### DISCUSSION TITLE
Polarity per app

### DISCUSSION THREAD

**Discussion by Julien76** [nix-community/stylix#396] [Q&A] [answered]
Hi, 
I am reading the documentation but it is not obvious to me if we can apply a different polarity per app/module?
For example, I want to use catpuccin light and dark variant: light one for file explorer, firefox... dark one for terminal, code editor...
Is it possible?

**Comment by danth** [comment, marked as answer]
Unfortunately, this isn't possible yet. See #208

**Reply by Julien76** [reply]
Thank you for the answer and sorry for my late one :)
I will for now style everything but the terminal and editor with Stylix so they have a consistent "light" theming and do the rest via home-manager as I currently do.
